### DISCUSSION TITLE
Check your license status

### DISCUSSION THREAD

**Discussion by izhangzhihao** [izhangzhihao/intellij-rainbow-brackets#2630] [Show and tell]
In this article, I will guide you on how to check the status of your license and what actions to take if your license is not activated.

If you encounter any issues with license activation, this article will provide you with the necessary steps to troubleshoot the problem. 

By following these guidelines, you can ensure that your license is up-to-date and that you can continue to use your software without any interruptions.

You can check the license under <kbd>Settings/Preferences</kbd> > <kbd>Other Settings</kbd> > <kbd>Rainbow Brackets</kbd>

- In a normal situation, you will see: 

<img width="1149" height="834" alt="image" src="https://github.com/user-attachments/assets/e1d4dc5a-edbf-4dea-91b0-a6af83ee19f8" />

- Or you may didn't have a valid license yet, you will see:

<img width="1094" height="834" alt="image" src="https://github.com/user-attachments/assets/1d7990ea-b82f-4167-9614-dd48e80138b9" />

- Or you do have a valid license, but the IDE is hasn't realized yet

<img width="1094" height="834" alt="image" src="https://github.com/user-attachments/assets/e1475789-12d5-4070-83b2-19dc5f80ff36" />

***You can wait a minute or refresh your license***: https://github.com/izhangzhihao/intellij-rainbow-brackets/discussions/2603#discussioncomment-5199473

- If you see the below message, means that you might be using a cracked version of this plugin or IDE.

<img width="1094" height="834" alt="image" src="https://github.com/user-attachments/assets/395f7baa-d95a-44ad-896d-3315981d9843" />

- If you want to delete the ja-netfilter, pls follow:

to remove JanetFilter using its uninstall [scripts](https://1drv.ms/f/c/d0eebbebdb9a76ac/EmNQDW1xA2BClkPBFvGcoe8BudVJ-a9LPdcQ-YvF68dSQQ?e=ihdTfy)

macOS or Linux: execute `mac-linux-uninstall-ja-netfilter.sh`
Windows: double click to execute `windows-uninstall-ja-netfilter.vbs`

- refresh your license status:
<img width="544" alt="image" src="https://github.com/izhangzhihao/intellij-rainbow-brackets/assets/12044174/d7cf738d-023b-402e-b6db-20feef7a306f">

<img width="536" alt="image" src="https://github.com/izhangzhihao/intellij-rainbow-brackets/assets/12044174/a85de4fd-95d3-456f-af3b-756ae1c4c9ce">

**Comment by JonnyJiang123** [comment]
![image](https://github.com/izhangzhihao/intellij-rainbow-brackets/assets/56102991/4019f902-992b-4f71-b5b0-7e7f5734201f)
这个情况意味着不能使用其他软件破解的idea了。必须使用自己激活idea才能激活Rainbow插件？

**Reply by JonnyJiang123** [reply]
![image](https://github.com/izhangzhihao/intellij-rainbow-brackets/assets/56102991/bf42882e-558d-4a50-9838-0061a4c769a9)

**Reply by izhangzhihao** [reply]
是的，IDE 被破解往往也意味着插件被破解，所以如果 IDE 都被破解了，插件是不会认为自己被真正激活了。

**Reply by izhangzhihao** [reply]
针对你这种情况，建议优先使用社区版或者 [EAP 版本](https://www.jetbrains.com/idea/nextversion/)等不需要 license 的 IDE。

**Reply by Poeeeeeee** [reply]
idea没有破解并且已经订阅了，但是还是显示我是破解的状态。可能是因为我的pycharm是破解的，然后我把pycharm卸载了，破解工具也卸载了，rawbow插件卸载后又重新下载，还是显示我是破解的，不知道你们是怎么检测是否破解的。我现在订阅了一个月插件但是却不能使用付费功能。

**Reply by izhangzhihao** [reply]
> idea没有破解并且已经订阅了，但是还是显示我是破解的状态。可能是因为我的pycharm是破解的，然后我把pycharm卸载了，破解工具也卸载了，rawbow插件卸载后又重新下载，还是显示我是破解的，不知道你们是怎么检测是否破解的。我现在订阅了一个月插件但是却不能使用付费功能。

可以邮件我 license 状态截图，和你的插件列表和你的 vmoptions 吗

**Reply by Poeeeeeee** [reply]
重装IDE后解决了

**Comment by zhh518** [comment]
![QQ图片20240117092556](https://github.com/izhangzhihao/intellij-rainbow-brackets/assets/38120264/4712fe2b-9a89-437f-be40-11f546640c72)

**Reply by izhangzhihao** [reply]
If you want to delete the ja-netfilter, pls follow:
in search everywhere, input vmoptions click on, edit/delete ja netfilter related info, save the file, and restart.
Or you may installed another plugin built by cracker, pls check https://github.com/izhangzhihao/intellij-rainbow-brackets/discussions/2696

**Comment by DEAN-Cherry** [comment]
![图片](https://github.com/izhangzhihao/intellij-rainbow-brackets/assets/76607677/d40bf559-bb74-4a8b-b509-93e626d29b3d)
这种情况是什么问题呢？
我激活了但是始终不生效，我购买的的 lifetime license

**Reply by izhangzhihao** [reply]
这种情况可能是 IDE 刚刚启动，license 还未初始化结束，如果是这个原因就等待一会儿就行了。
还可能是 IDE 正在进行indexing， 需要等待index 结束编辑器才会刷新，这种情况可以观察 IDE 右下角的task是否结束了，结束后可以关闭当前文件再重新打开以实现手动刷新的目的试试看。
如果还是不行，可以邮件给我你的 idea log。

**Comment by david-bakin-sl** [comment]
Using IntelliJ IDEA 2024.1 (ultimate) on a company license.  Rainbow brackets license page now looks like:

![image](https://github.com/izhangzhihao/intellij-rainbow-brackets/assets/117694041/4ca1458b-4bf9-400f-8918-d656a944409c)

I have a Personal License that should be valid through 16 June 2024 - this license I bought with my JetBrains account but activated in the company-licensed instance of IntelliJ.  It _used_ to work fine (activated Rainbow Brackets, the paid features could be selected, and they worked.  I actually don't know when it _stopped_ working, but as you can see, IntelliJ now thinks there is no valid license.

I cannot find where to find the "refresh license list" button mentioned above.

Also, if I click, in the PDF of the license itself, the link to "proceed with software activation" (which points to somewhere under `account.jetbrains.com/order/assets` I see the following from JetBrains:

![image](https://github.com/izhangzhihao/intellij-rainbow-brackets/assets/117694041/e9d0f87d-a6c0-4b4d-b7fe-f201fca6fef8)

How to proceed?  Thank you!

**Reply by izhangzhihao** [reply]
You can use your personal account’ offline code with your corporate account by : 
https://sales.jetbrains.com/hc/en-gb/articles/360016995379-Activating-JetBrains-IDEs-with-an-offline-activation-code

**Comment by vladyn** [comment]
Hello there, 
I have purchased license on my email from the JB store. I have though a trial period of usage of me Webstorm. My question is - can I use my paid version of the Rainbow brackets before my next Webstorm license purchase?

**Reply by izhangzhihao** [reply]
Yes

**Reply by vladyn** [reply]
How?

**Reply by izhangzhihao** [reply]
No special actions are required.The paid version of Rainbow Brackets can be used with the trial version of WebStorm IDE, as long as your IDE is not illegally cracked.

**Comment by Osanosa** [comment]
![image](https://github.com/user-attachments/assets/6000bb05-03ce-48a1-9d65-7e73b74e77e6)  
![image](https://github.com/user-attachments/assets/d54da768-3c28-4960-a247-98540e9f422c)  
why

**Reply by izhangzhihao** [reply]
Look at the end of your configuration file, \Users\Osanosa\Downloads\Compressed\jetbra\ja-netfilter.jar=jetbrains, you are using ja-netfilter to crack. If you have purchased a license, you can reinstall the IDE and stop using ja-netfilter.

**Reply by Osanosa** [reply]
Thanks. Didn't notice that. I don't use that, I just wanted to look at WebStorm a few months ago and installed it. Never actually used it. And also it became free after. So I have no idea how to uninstall it

**Reply by izhangzhihao** [reply]
just uninstall your IDE and reinstall it from JetBrains.

**Comment by draskosaric** [comment]
Today I bought a lifetime licence

<img width="905" height="991" alt="Screenshot 2026-03-04 at 18 45 05" src="https://github.com/user-attachments/assets/4d518fe9-6c1d-4885-b4c9-0a6b021c6d49" />

And in Android Studio it says that it is valid only one year!

<img width="702" height="281" alt="Screenshot 2026-03-04 at 18 47 49" src="https://github.com/user-attachments/assets/7e59bd29-1548-457e-9dbe-f3359ea4b74e" />

Configuration and info are below:

--- RainbowBrackets Settings ---
<RainbowSettings>
  <option name="applyColorsOfRoundForAllBrackets" value="false" />
  <option name="bigFilesLinesThreshold" value="5000" />
  <option name="borderHighlightingStrength" value="0.5" />
  <option name="colorGenHueOption" value="random" />
  <option name="colorGenLuminosityOption" value="auto" />
  <option name="customIndentGuides" value="false" />
  <option name="cycleCountOnAllBrackets" value="false" />
  <option name="disableRainbowIndentsInZenMode" value="true" />
  <option name="disableReaderMode" value="false" />
  <option name="doNOTRainbowifyBigFiles" value="false" />
  <option name="doNOTRainbowifyBracketsWithoutContent" value="false" />
  <option name="doNOTRainbowifyTemplateString" value="false" />
  <option name="doNOTRainbowifyTheFirstLevel" value="false" />
  <option name="enableRainbowAngleBrackets" value="true" />
  <option name="enableRainbowRoundBrackets" value="true" />
  <option name="enableRainbowSquareBrackets" value="true" />
  <option name="enableRainbowSquigglyBrackets" value="true" />
  <option name="enableRainbowVariables" value="false" />
  <option name="enableRainbowVariablesBySemanticHighlighting" value="false" />
  <option name="highlightCurrentBlockOptions" value="None" />
  <option name="indentGuideLineHighlightingStrength" value="0.3" />
  <option name="indentGuidesOptions" value="Only selected indent guide" />
  <option name="languageBlacklist">
    <set>
      <option value="hocon" />
      <option value="mxml" />
      <option value="Twig" />
    </set>
  </option>
  <option name="numberOfColors" value="5" />
  <option name="overrideMatchedBraceAttributes" value="true" />
  <option name="pressAnyKeyToRemoveTheHighlightingEffects" value="false" />
  <option name="rainbowEnabled" value="true" />
  <option name="rainbowVariablesOptionShowed" value="false" />
  <option name="rainbowifyHTMLInsideJS" value="true" />
  <option name="rainbowifyKotlinFunctionLiteralBracesAndArrow" value="true" />
  <option name="rainbowifyKotlinLabel" value="true" />
  <option name="rainbowifyPythonKeywords" value="false" />
  <option name="rainbowifyRubyKeywords" value="false" />
  <option name="rainbowifyTagNameInXML" value="true" />
  <option name="rainbowifyTagNameInYAML" value="true" />
  <option name="restrainHighlightingBGColor" value="#808080" />
  <option name="restrainHighlightingFGColor" value="#808080" />
  <option name="restrainHighlightingKeepOriginalColor" value="false" />
  <option name="restrainHighlightingStrength" value="0.45" />
  <option name="scopeHighlightStrength" value="0.2" />
  <option name="semanticHighlightingList">
    <set>
      <option value="C#" />
      <option value="F#" />
      <option value="Dart" />
      <option value="FreeMarker" />
      <option value="C++" />
    </set>
  </option>
  <option name="showNotificationOnUpdate" value="true" />
  <option name="showRainbowIndentGuides" value="true" />
  <option name="suppressAdvancedIndentGuidesBanner" value="false" />
  <option name="suppressBigFileCheck" value="false" />
  <option name="suppressBlackListCheck" value="false" />
  <option name="suppressDisabledCheck" value="false" />
  <option name="suppressLicenseBanner" value="false" />
  <option name="suppressRubyCheck" value="false" />
  <option name="suppressXMLCheck" value="false" />
  <option name="suppressYAMLCheck" value="false" />
  <option name="useColorGenerator" value="false" />
  <option name="useColorGeneratorForRainbowVariables" value="false" />
  <option name="version" value="2025.3.7" />
</RainbowSettings>

--- Environment Info ---
License Status: valid
Plugin Version: 2025.3.7
IDE Version: Narwhal 3 Feature Drop | 2025.1.3
IDE Build: AI-251.26094.121.2513.14007798
OS: Mac OS X (26.3)
Max Memory: 7168M
Free Memory: 177M
Cores: 12

Non Bundled Plugins: 
Dart: 251.27623.5
Sexy Editor: 5.2.0
com.anthropic.code.plugin: 0.1.14-beta
com.bloc.intellij_generator_plugin: 4.1.11
com.intellij.marketplace: 251.26094.157
com.ixigua.flutter_asset_iteral: 1.7.8
com.localizely.flutter-intl: 1.18.8-2024.2
com.redhat.devtools.lsp4ij: 0.19.2
io.flutter: 87.1
izhangzhihao.rainbow.brackets: 2025.3.7

--- vmoptions ---
-Xmx7168m

**Reply by izhangzhihao** [reply]
Hi thanks for your purchase

It is as expected and the license will renew automatically every year. No action needed from your side. Just enjoy it.

**Comment by decafbad** [comment]
Hi. I bought a yearly license yesterday, it was working on IntelliJ. But I moved to WebStorm now and won't use IntelliJ again.
My ID on https://account.jetbrains.com/licenses page is BG9ZBT24J3. I couldn't find a way to activate the plugin as premium.

**Reply by izhangzhihao** [reply]
Can you share the info by clicking "Copy Configuration and info" from the setting page?

**Comment by decafbad** [comment]
It's licensed now, works. Thanks.

**Comment by ghost** [comment]
# Issue

The plugin activation script provided by ckey.run activated the IDE and plugins, but the plugin settings page shows the plugin is not activated.

<img width="1389" height="928" alt="1" src="https://github.com/user-attachments/assets/d3d87174-6906-482f-a7a7-cd4063e2d594" />
<img width="1424" height="930" alt="2" src="https://github.com/user-attachments/assets/0f58d1a2-af3a-4ff0-9d0d-9edcd9565f14" />

# Configuration and Information

--- RainbowBrackets Settings ---
<RainbowSettings>
  <option name="applyColorsOfRoundForAllBrackets" value="false" />
  <option name="bigFilesLinesThreshold" value="5000" />
  <option name="borderHighlightingStrength" value="0.5" />
  <option name="colorGenHueOption" value="random" />
  <option name="colorGenLuminosityOption" value="auto" />
  <option name="customIndentGuides" value="false" />
  <option name="cycleCountOnAllBrackets" value="false" />
  <option name="disableRainbowIndentsInZenMode" value="true" />
  <option name="disableReaderMode" value="false" />
  <option name="doNOTRainbowifyBigFiles" value="true" />
  <option name="doNOTRainbowifyBracketsWithoutContent" value="false" />
  <option name="doNOTRainbowifyTemplateString" value="false" />
  <option name="doNOTRainbowifyTheFirstLevel" value="false" />
  <option name="enableRainbowAngleBrackets" value="true" />
  <option name="enableRainbowRoundBrackets" value="true" />
  <option name="enableRainbowSquareBrackets" value="true" />
  <option name="enableRainbowSquigglyBrackets" value="true" />
  <option name="enableRainbowVariables" value="false" />
  <option name="enableRainbowVariablesBySemanticHighlighting" value="false" />
  <option name="highlightCurrentBlockOptions" value="None" />
  <option name="indentGuideLineHighlightingStrength" value="0.3" />
  <option name="indentGuidesOptions" value="Only selected indent guide" />
  <option name="languageBlacklist">
    <set>
      <option value="hocon" />
      <option value="mxml" />
      <option value="Twig" />
    </set>
  </option>
  <option name="numberOfColors" value="10" />
  <option name="overrideMatchedBraceAttributes" value="true" />
  <option name="pressAnyKeyToRemoveTheHighlightingEffects" value="false" />
  <option name="rainbowEnabled" value="true" />
  <option name="rainbowVariablesOptionShowed" value="false" />
  <option name="rainbowifyCMacroKeywords" value="false" />
  <option name="rainbowifyHTMLInsideJS" value="true" />
  <option name="rainbowifyKotlinFunctionLiteralBracesAndArrow" value="true" />
  <option name="rainbowifyKotlinLabel" value="true" />
  <option name="rainbowifyPythonKeywords" value="false" />
  <option name="rainbowifyRubyKeywords" value="false" />
  <option name="rainbowifyTagNameInXML" value="false" />
  <option name="rainbowifyTagNameInYAML" value="false" />
  <option name="restrainHighlightingBGColor" value="#808080" />
  <option name="restrainHighlightingFGColor" value="#808080" />
  <option name="restrainHighlightingKeepOriginalColor" value="false" />
  <option name="restrainHighlightingStrength" value="0.45" />
  <option name="scopeHighlightStrength" value="0.2" />
  <option name="semanticHighlightingList">
    <set>
      <option value="C#" />
      <option value="F#" />
      <option value="Dart" />
      <option value="FreeMarker" />
      <option value="C++" />
    </set>
  </option>
  <option name="showNotificationOnUpdate" value="true" />
  <option name="showRainbowIndentGuides" value="true" />
  <option name="suppressAdvancedIndentGuidesBanner" value="false" />
  <option name="suppressBigFileCheck" value="false" />
  <option name="suppressBlackListCheck" value="false" />
  <option name="suppressDisabledCheck" value="false" />
  <option name="suppressLicenseBanner" value="false" />
  <option name="suppressRainbowifyCMacroKeywords" value="false" />
  <option name="suppressRainbowifyPuml" value="false" />
  <option name="suppressRubyCheck" value="false" />
  <option name="suppressScopeHighlightingWhenDebugging" value="false" />
  <option name="suppressScopeHighlightingWhenDiffing" value="false" />
  <option name="suppressXMLCheck" value="false" />
  <option name="suppressYAMLCheck" value="false" />
  <option name="useColorGenerator" value="false" />
  <option name="useColorGeneratorForRainbowVariables" value="false" />
  <option name="version" value="2025.3.9" />
</RainbowSettings>

--- Environment Info ---
License Status: No valid license found: AwGgjCpZlA==
Plugin Version: 2025.3.9
IDE Version: 2026.1.1
IDE Build: PY-261.23567.174
OS: Windows 11 (10.0)
Max Memory: 2048M
Free Memory: 191M
Cores: 32

Non Bundled Plugins: 
Keymap exporter: 2.7
cn.yiiguxing.plugin.translate: 3.8.3
com.alayouni.ansiHighlight: 26.1.1
com.andrey4623.rainbowcsv: 2.0.2
com.chylex.intellij.inspectionlens: 1.6.1
com.clutcher.comments_highlighter: 2025.3.4
com.firsttimeinforever.intellij.pdf.viewer.intellij-pdf-viewer: 0.18.0
com.github.peppy.ppyjetbrainstheme: 0.0.6
com.intellij.plugins.vscodekeymap: 261.22158.180
com.mallowigi: 201.0.1
com.sburlyaev.terminal.plugin: 0.8.1
com.shuzijun.markdown-editor: 2.0.5
de.netnexus.camelcaseplugin: 3.0.12
izhangzhihao.rainbow.brackets: 2025.3.9
me.fornever.commandlink: 1.1.0
mobi.hsz.idea.gitignore: 4.5.7
net.bluehill.commentRemover: 0.3.1
net.seesharpsoft.intellij.plugins.csv: 4.2.0
org.exbin.deltahex.intellij: 0.2.13
org.jetbrains.completion.full.line: 261.23567.182
org.sylfra.idea.plugins.linessorter: 1.0.1
org.tonybaloney.security.pycharm-security: 1.29.0
pl.sudo.imageviewer.python: 4.3.5
ru.adelf.idea.dotenv: 261.23567.176
ru.meanmail.plugins.prettify-python: 2025.2
zielu.gittoolbox: 600.2.6+253

--- vmoptions ---

**Reply by izhangzhihao** [reply]
It looks like you are reporting an issue caused by a third-party cracking tool. Please note that I do not support pirated versions of my plugin.

Cracks do not work: The software has protections in place. Using a crack will break the core functionality of the app.

Security risks: Cracking tools are notorious for carrying viruses, trojans, and malware. You are putting your computer and data at immense risk.

Support genuine work: I have been out of a job for months, and the small revenue from this plugin is literally what keeps me afloat right now.

Engaging in piracy hurts independent developers and ruins the community. Please respect the effort that goes into creating this plugin and support the genuine version.

I saw your original comment is Chinese, If you wish to support, pls visit [here](https://docs.qq.com/doc/DQ3NaT1NJVUJka2h0).

**Comment by jlfsdtc** [comment]
After upgrading to version 2026.2, the plugin displays the notification "no valid license found".
<img width="1014" height="430" alt="image" src="https://github.com/user-attachments/assets/8b308ff7-e379-4ede-91b0-c20e31ca8bc3" />
<img width="2630" height="1688" alt="image" src="https://github.com/user-attachments/assets/e7764001-55e5-43d9-ad5b-c904a76bcec4" />

**Reply by jlfsdtc** [reply]
However, it functions normally in version 2026.1.

**Reply by izhangzhihao** [reply]
Please stop using ja-netfilter to crack the plugin and the IDE.  

Some cracking methods may work for a short time, but they will never be effective long-term. If you respect genuine software, please remove the ja-netfilter-related Java agent properties from the `vmoptions` file, following the instructions under the red text in the image.

**Reply by izhangzhihao** [reply]
If you believe you have not used any cracking tools, please click the **Copy config and info** button and send the copied content to me.

**Reply by jlfsdtc** [reply]
--- RainbowBrackets Settings ---
<RainbowSettings>
  <option name="applyColorsOfRoundForAllBrackets" value="false" />
  <option name="bigFilesLinesThreshold" value="5000" />
  <option name="borderHighlightingStrength" value="0.5" />
  <option name="colorGenHueOption" value="random" />
  <option name="colorGenLuminosityOption" value="auto" />
  <option name="customIndentGuides" value="false" />
  <option name="cycleCountOnAllBrackets" value="false" />
  <option name="disableRainbowIndentsInZenMode" value="true" />
  <option name="disableReaderMode" value="false" />
  <option name="doNOTRainbowifyBigFiles" value="false" />
  <option name="doNOTRainbowifyBracketsWithoutContent" value="false" />
  <option name="doNOTRainbowifyTemplateString" value="false" />
  <option name="doNOTRainbowifyTheFirstLevel" value="false" />
  <option name="enableRainbowAngleBrackets" value="true" />
  <option name="enableRainbowRoundBrackets" value="true" />
  <option name="enableRainbowSquareBrackets" value="true" />
  <option name="enableRainbowSquigglyBrackets" value="true" />
  <option name="enableRainbowVariables" value="true" />
  <option name="enableRainbowVariablesBySemanticHighlighting" value="false" />
  <option name="highlightCurrentBlockOptions" value="None" />
  <option name="indentGuideLineHighlightingStrength" value="0.3" />
  <option name="indentGuidesOptions" value="Only selected indent guide" />
  <option name="languageBlacklist">
    <set>
      <option value="hocon" />
      <option value="mxml" />
      <option value="Twig" />
    </set>
  </option>
  <option name="numberOfColors" value="9" />
  <option name="overrideMatchedBraceAttributes" value="true" />
  <option name="pressAnyKeyToRemoveTheHighlightingEffects" value="false" />
  <option name="rainbowEnabled" value="true" />
  <option name="rainbowVariablesOptionShowed" value="false" />
  <option name="rainbowifyCMacroKeywords" value="false" />
  <option name="rainbowifyHTMLInsideJS" value="true" />
  <option name="rainbowifyKotlinFunctionLiteralBracesAndArrow" value="true" />
  <option name="rainbowifyKotlinLabel" value="true" />
  <option name="rainbowifyPythonKeywords" value="true" />
  <option name="rainbowifyRubyKeywords" value="true" />
  <option name="rainbowifyTagNameInXML" value="true" />
  <option name="rainbowifyTagNameInYAML" value="true" />
  <option name="restrainHighlightingBGColor" value="#808080" />
  <option name="restrainHighlightingFGColor" value="#808080" />
  <option name="restrainHighlightingKeepOriginalColor" value="false" />
  <option name="restrainHighlightingStrength" value="0.45" />
  <option name="scopeHighlightStrength" value="0.2" />
  <option name="semanticHighlightingList">
    <set>
      <option value="C#" />
      <option value="F#" />
      <option value="Dart" />
      <option value="FreeMarker" />
      <option value="C++" />
      <option value="SQL" />
    </set>
  </option>
  <option name="showNotificationOnUpdate" value="true" />
  <option name="showRainbowIndentGuides" value="true" />
  <option name="suppressAdvancedIndentGuidesBanner" value="false" />
  <option name="suppressBigFileCheck" value="false" />
  <option name="suppressBlackListCheck" value="false" />
  <option name="suppressDisabledCheck" value="false" />
  <option name="suppressLicenseBanner" value="false" />
  <option name="suppressRainbowifyCMacroKeywords" value="false" />
  <option name="suppressRainbowifyPuml" value="false" />
  <option name="suppressRubyCheck" value="false" />
  <option name="suppressScopeHighlightingWhenDebugging" value="false" />
  <option name="suppressScopeHighlightingWhenDiffing" value="false" />
  <option name="suppressXMLCheck" value="false" />
  <option name="suppressYAMLCheck" value="false" />
  <option name="useColorGenerator" value="false" />
  <option name="useColorGeneratorForRainbowVariables" value="false" />
  <option name="version" value="2026.2.0" />
</RainbowSettings>

--- Environment Info ---
License Status: Oops, no valid license found.: AwGl7TiA
Plugin Version: 2026.2.0
IDE Version: 2026.2.0.1
IDE Build: IU-262.8665.337
OS: Mac OS X (26.6)
Max Memory: 16384M
Free Memory: 12778M
Cores: 18

Non Bundled Plugins: 
EclipseCodeFormatter: 23.5.241.000.0-Eclipse_2024-09
GradleDependenciesHelper: 1.16
GrepConsole: 13.3.0-IJ2023.3
HighlightBracketPair: 1.5.5
MavenRunHelper: 5.1.0-IJ2026.1
PerforceDirectPlugin: 262.8665.176
PlantUML integration: 7.14.0-IJ2023.2
SequenceDiagram: 4.0.8
Statistic: 5.2
Subversion: 262.8665.369
bot.cline: 1.1.64
cn.yiiguxing.plugin.translate: 3.8.4
com.ccnode.codegenerator.MyBatisCodeHelperPro: 3.5.8+2321
com.codelens.pro: 1.0.6
com.github.copilot: 1.14.2-261-macos-arm64
com.github.goldsubmarine.restfulhelper: 0.4.7-stable
com.github.oowekyala.javacc: 1.12.3
com.github.platan.gradle-dependencies-formatter: 0.6.1
com.intellij.jvm.dfa.analysis: 262.8665.176
com.intellij.nativeDebug: 262.8665.176
com.intellij.notebooks.core: 262.8665.270
com.intellij.reactivestreams: 262.8665.176
com.intellij.spring.debugger: 262.8665.176
com.intellij.tasks: 262.8665.173
com.jetbrains.rust: 262.8665.323
com.mallowigi: 201.0.1
com.nasller.CodeGlancePro: 2.0.8
de.ax.powermode: 105.501
de.netnexus.camelcaseplugin: 3.0.12
intellij.grid.loader.parquet: 262.8665.173
intellij.jupyter: 262.8665.270
io.github.godfather1103.alibaba.p3c: 2.3-2026.1-2.1.1-ext-6
izhangzhihao.rainbow.brackets: 2026.2.0
leetcode-editor: 8.16
mobi.hsz.idea.gitignore: 4.5.7
org.intellij.scala: 2026.2.15
org.jetbrains.plugins.ruby: 262.8665.309
some.awesome: 1.16

--- vmoptions ---
-XX:+UseZGC
-Xms16g
-Xmx16g
--add-opens=java.base/jdk.internal.org.objectweb.asm=ALL-UNNAMED
--add-opens=java.base/jdk.internal.org.objectweb.asm.tree=ALL-UNNAMED
-Dsbt.override.build.repos=true
-Dide.managed.by.toolbox=/Applications/JetBrains Toolbox.app/Contents/MacOS/jetbrains-toolbox
-Dtoolbox.notification.portFile=/Users/phoenix/Library/Caches/JetBrains/Toolbox/ports/3db12439-ab88-4352-b025-df955351221f.port

**Reply by izhangzhihao** [reply]
remove it from your vmoptions file

--add-opens=java.base/jdk.internal.org.objectweb.asm=ALL-UNNAMED
--add-opens=java.base/jdk.internal.org.objectweb.asm.tree=ALL-UNNAMED

**Reply by jlfsdtc** [reply]
After deleting the `CodeLens Pro` plugin, the `rainbow-brackets` plugin started working normally again. I suspect there was a conflict between the plugins.

**Comment by hellooldtimes** [comment]
大佬，怎么用免费版？

**Reply by jlfsdtc** [reply]
Just install it directly from the plugin marketplace; I only paid after using it for a while.
